scripts/benchmark_modules_on_android.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Benchmarks modules generating by compile_modules.sh on Android and writes
+# performance data to a text proto file, perfgate-SERIAL_NUMBER-GIT_HASH.
+#
+# The script benchmarks modules on 7-th big core, ie, run with `taskset 80`.
+
+set -o errexit
+set -o pipefail
+
+GIT_HASH="UNKNOWN"
+while [[ $# -gt 0 ]]; do
+  token="$1"
+  case $token in
+    --model=*)
+      MODEL=${1#*=}
+      shift
+      ;;
+    --targets=*)
+      TARGETS=${1#*=}
+      shift
+      ;;
+    --benchmark_key=*)
+      BENCHMARK_KEY=${1#*=}
+      shift
+      ;;
+    --git_hash=*)
+      GIT_HASH=${1#*=}
+      shift
+      ;;
+  esac
+done
+
+TS=$(date +%s)000
+SED_EXPR='s/BM_[[:alnum:][:punct:]]+ [[:space:]]+ ([0-9]+) ms[[:print:]]+/\1/p'
+OUTPUT_DIR=test_output_files
+DEVICE_ROOT=/data/local/tmp/benchmark_tmpdir
+
+mkdir -p "${OUTPUT_DIR}"
+
+if [[ -z "${MODEL}" ]]; then
+  echo "Must set --model flag.";
+  exit 1
+fi
+
+if [[ -z "${BENCHMARK_KEY}" ]]; then
+  echo "Must set --benchmark_key flag.";
+  exit 1
+fi
+
+function perfgate_metadata {
```

Comment:
1) replace with mako
2) rename with append_mako_*

I think the output redirection would make it a bit redundant because we will capture the result and do something like `echo "$(foo)" >> bar.txt`. What do you think?

The below snippet is my prototype.
```bash
function foo {
  cat << EOF
metadata: {
  git_hash:
  timestamp_ms:
  benchmark_key:
}
EOF
}

echo "$(foo)" >> bar.txt
```